include/drivers/pio/sunxi-pio.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_PIO_SUNXI_PIO_H
+#define DRIVERS_PIO_SUNXI_PIO_H
+
+#define SUNXI_PIO_PIN(port, index) ((port) << 5 | ((index) & 0x1f))
```

Comment:
This could also use `BITMASK(0, 5)` instead of `0x1f` if you think it makes it more clear

Reply:
I do. Updated.